match/lib/match/runner.rb

```diff
@@ -197,7 +197,7 @@ def fetch_provisioning_profile(params: nil, certificate_id: nil, app_identifier:
       parsed = FastlaneCore::ProvisioningProfile.parse(profile, keychain_path)
       uuid = parsed["UUID"]
 
-      if spaceship && !spaceship.profile_exists(username: params[:username], uuid: uuid)
+      if spaceship && !spaceship.profile_exists(username: params[:username], uuid: uuid, return_if_missing: params[:recreate_deleted_profiles])
```

Comment:
The "renaming" of the parameter here from `recreate_deleted_profiles` to `return_if_missing` adds unnecessary complexity, can we keep the name?